flexmeasures/api/v3_0/assets.py

```diff
@@ -786,3 +786,53 @@ def get_jobs(self, id: int, asset: GenericAsset):
             "jobs": all_jobs_data,
             "redis_connection_err": redis_connection_err,
         }, 200
+
+    @route("/default_asset_view", methods=["GET"])
+    @permission_required_for_context("read", ctx_loader=AccountIdField.load_current)
+    @as_json
+    def update_default_asset_view(self):
+        """Update the default asset view for the current user.
+
+        .. :quickref: Asset; Update the default asset view
+
+        **Example request**
+
+        .. sourcecode:: json
+
+            {
+                "default_asset_view": "Graphs",
+                "set_default": true
```

Comment:
I believe the name `use_as_default` is better, as it also is used to unset the setting.

Also use `@use_kwargs`. it's good for readability.